src/js/popup.js

```diff
@@ -247,9 +263,32 @@ function init() {
     visibility: 'visible'
   });
 
+  let link = getLink();
+  $("#cta-link").attr("href", link.url);
+  $('#cta-text').html(chrome.i18n.getMessage(link.text));
+  $('#cta-icon').addClass(link.icon);
+
   window.POPUP_INITIALIZED = true;
 }
 
+/**
+ * Get the link to display at the bottom of the popup
+ */
+function getLink() {
```

Comment:
Have this function live where `linkRotation` is now, and have `linkRotation` live inside the function? We don't need the var outside.